src/vellum/workflows/workflows/base.py

```diff
@@ -110,9 +110,10 @@ def collect_nodes(graph_item: Union[GraphAttribute, Set[GraphAttribute]]) -> Set
                         nodes.update(node for node in item.nodes)
                     elif inspect.isclass(item) and issubclass(item, BaseNode):
                         nodes.add(item)
-            elif issubclass(graph_item, BaseNode):
+            elif inspect.isclass(graph_item) and issubclass(graph_item, BaseNode):
                 nodes.add(graph_item)
             else:
+                # We should never get here, but just in case
```

Comment:
SDK first users could actually get here by doing dumb stuff like:
```python
class Workflow(BaseWorkflow):
    graph = 5
```

which is why I like us now falling through to a user facing error case here. Let's solidify with a quick unit test!